core/src/main/java/io/questdb/cairo/wal/WalTxnType.java

```diff
@@ -26,7 +26,13 @@
 
 public class WalTxnType {
     public static final byte DATA = 0;
+    public static final byte MAT_VIEW_DATA = 3;
+    public static final byte MAT_VIEW_INVALIDATE = 4;
```

Comment:
Idea, possibly overkill:

Would it make sense to reserve a bit range for mat view events?
Then if by chance we find that we need more events in the next release there's more flexibility?

Something like:
```java
public class WalTxnType {
    public static final byte MAT_VIEW_FLAG = (byte) -128;
    public static final byte DATA = 0;
    public static final byte MAT_VIEW_DATA = 1 | MAT_VIEW_FLAG;
    public static final byte MAT_VIEW_INVALIDATE = 2 | MAT_VIEW_FLAG;
```

Then have the next "bridge" release ignore any event that has the `MAT_VIEW_FLAG`?